### DISCUSSION TITLE
Creating a submenu in egui::menu::menu

### DISCUSSION THREAD

**Discussion by crumblingstatue** [emilk/egui#739] [Q&A] [answered]
I want a submenu for a recently used files list.
I tried `CollapsingHeader`, but the whole menu closes if the user clicks on an item.

**Comment by emilk** [comment, marked as answer]
This is not yet possible, but soon will be once https://github.com/emilk/egui/pull/543 is merged!

**Reply by crumblingstatue** [reply]
That's very cool, thank you!
